README.md

```diff
@@ -46,3 +46,5 @@ Unzipped the full parquet data will be approximately 70GB. Each (compressed) chu
 - [This](https://github.com/brandon-rhodes/pycon-pandas-tutorial) is a great tutorial to learn Pandas.
 
 - [Tutorial](https://www.youtube.com/watch?v=HW29067qVWk) on Jupyter Notebook.
+
+- We have used dask in some of our Jupyter notebooks. Dask exposes lower-level APIs letting you build custom systems for in-house applications. It also makes it working with larger datasets easier. Please start by reviewing the [docs](https://dask.org/) to learn more about it.
```

Comment:
>  Dask exposes lower-level APIs letting you build custom systems for in-house applications.

This sentence is taken from the dask landing page, but is out of context and not relevant to this project. I think the simplest and most relevant way of describing dask is that it gives you a pandas-like API (it often looks like pandas but not every pandas function is available) but lets you work on data that is too big to fit in memory. Dask can be used on a single machine or a cluster. Most analyses done for this project were done on a single machine.